from ..errors import DomainError


class ConnectionNotFoundError(DomainError):
    pass
